src/api/comment.tsx

```diff
@@ -0,0 +1,86 @@
+import { ApiResponse } from "apisauce";
+import api from "./base";
+import {
+  fetchPostCommentsApiPayload,
+  fetchPostCommentsApiResponse,
+  createPostCommentApiResponse,
+  createPostCommentApiPayload,
+  updatePostCommentApiResponse,
+  updatePostCommentApiPayload,
+  deletePostCommentApiResponse,
+  deletePostCommentApiPayload,
+} from "./commentType";
+
+/**
+ * Fetch all comments based on postId
+ */
+const fetchPostComments = ({
+  payload,
+}: fetchPostCommentsApiPayload): Promise<
+  ApiResponse<fetchPostCommentsApiResponse>
+> => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Can move this to the top of the file and change it to a multi-line comment so you don't have to repeat it everytime

Reply:
resolved